package.json

```diff
@@ -1,6 +1,6 @@
 {
   "name": "@hulu/roca",
-  "version": "0.2.1",
+  "version": "0.2.2",
```

Comment:
Since we're adding a new backwards-compatible argument, let's make this a minor-version bump!  https://semver.org/#summary